docs/production-deployment/cloud/nexus/operations.mdx

```diff
@@ -0,0 +1,207 @@
+
+
+## Metrics
+
+Scheduling and processing a Nexus Operation is reported through existing cloud metrics with the following operation metric labels:
+
+- Caller Namespace
+  - RespondWorkflowTaskCompleted: This is used to schedule the Nexus Operation.
+- Handler Namespace
+  - PollNexusTaskQueue
+  - RespondNexusTaskCompleted
+  - RespondNexusTaskFailed
+
+See [Cloud Metrics](/production-deployment/cloud/metrics/reference) for additional details.
+
+## Audit Logging
+
+The following Nexus control plane actions are sent to the Audit Logging integration:
+
+- Create Nexus Endpoint: `CreateNexusEndpoint`
+- Update Nexus Endpoint: `UpdateNexusEndpoint`
+- Delete Nexus Endpoint: `DeleteNexusEndpoint`
+
+See [Audit Logging](/cloud/audit-logging) for additional details.
+
+## Rate Limiting
+
+Nexus requests (commands, polling) are counted as part of the overall Namespace RPS limit in both the caller and handler Namespaces.
+Default Namespace RPS limits are set at 1600 and automatically adjust based on recent usage (over prior 7 days).
+
+## SLOs & SLAs
+
+Nexus requests (commands, polling) have the same latency SLOs and error rate SLAs as other Worker requests in both the caller and handler Namespaces.
+
+See [Availability](/cloud/service-availability) and [SLA](/cloud/sla) for additional details.
+
+## Limits
+
+### Max Nexus Endpoints
+
+By default, each account is provisioned with a max of 10 Nexus Endpoints. 
+You can request further increases beyond the initial 10 Endpoint limit by opening a support ticket.
+
+### Workflow Max Nexus Operations
+
+A single Workflow Execution can have a maximum of 30 in-flight Nexus Operations and 30 total Nexus Operations (as Public Preview does not yet remove completed Nexus Operations from mutable state).
+After that limit is reached, no more Nexus Operations will be processed for that Workflow Execution.
+
+### Nexus Request Handler Timeout
+
+Nexus Operation handlers have less than 10 seconds to process a single Nexus start or cancel request.
+Handlers should observe the context deadline and ensure they do not exceed it.
+This includes fully processing a synchronous Nexus operation and starting an asynchronous Nexus operation, for example one that starts a Workflow.
+If a handler doesn’t respond within a context deadline, a context deadline exceeded error will be tracked in the caller Workflow’s pending Nexus operations, and the Nexus Machinery will retry the Nexus request with an exponential backoff policy.
+
+### Nexus Operation Maximum Duration
+
+Each Nexus Operation has a maximum ScheduleToClose duration of 60 days, which is most applicable to asynchronous Nexus Operations that are completed with an asynchronous callback using a separate Nexus request from the handler back to the caller Namespace. 
+The 60 day maximum is forward-compatible with completion callbacks that may be signed with a single use token such that asymmetric encryption keys may be rotated in a timely fashion. 
```

Comment:
as an end-user this sentence is somewhat confusing to me. I am guessing we are saying its a 60-day limit in order to give flexibility in case we want to make future modifications to the callback format prior to GA? Not sure if we want to re-word/change this up

Reply:
@bergundy

Reply:
"The 60 day maximum is a limit we will look to increase at some point in the future" might be better/easier to understand

Reply:
added